### DISCUSSION TITLE
A possible (or possibly impossible) addition

### DISCUSSION THREAD

**Discussion by Androxilogin** [sanni/cartreader#177] [Ideas & Suggestions]
Would it be possibly to add a script that would check the battery level of carts on screen?

**Comment by sanni** [comment]
The battery is not connected to the cart slot, so it's not possible.
It's voltage is only connected to the MAD1 and the SRAM chips.

![batterymad1](https://user-images.githubusercontent.com/19169494/122670122-99d91980-d1c0-11eb-8132-250e13aa1990.jpg)
